src/config.rs

```diff
@@ -90,6 +93,8 @@ impl Config {
             github_accesstoken: maybe_env("CRATESFYI_GITHUB_ACCESSTOKEN")?,
             github_updater_min_rate_limit: env("DOCSRS_GITHUB_UPDATER_MIN_RATE_LIMIT", 2500)?,
 
+            gitlab_accesstoken: maybe_env("CRATESFYI_GITLAB_ACCESSTOKEN")?,
```

Comment:
```suggestion
            gitlab_accesstoken: maybe_env("DOCSRS_GITLAB_ACCESSTOKEN")?,
```

Reply:
It's still using `CRATESFYI_GITHUB_ACCESSTOKEN` for the github token. Wouldn't it be more logical to keep the current name?

Reply:
I would really prefer for all new env variables to use DOCSRS_ instead. All the CRATESFYI stuff is legacy from years ago.

Reply:
Fine by me!